package.json

```
@@ -21,7 +21,8 @@
     "generate": "npm-run-all -s generate:ts-from-yaml:*",
     "generate:ts-from-yaml:api": "shx mkdir -p src/generated/payment_manager && shx mkdir -p src/generated/pagopa_proxy && rimraf src/generated/payment_manager/* && rimraf src/generated/pagopa_proxy/* && gen-api-models --strict 0 --api-spec api_payment_manager.yaml --out-dir src/generated/payment_manager --request-types && gen-api-models --strict 0 --api-spec https://raw.githubusercontent.com/pagopa/io-pagopa-proxy/v1.2.0/api-spec/api-pagopa-proxy.yaml --out-dir src/generated/pagopa_proxy --request-types",
     "generate:ts-from-yaml:pgs": "shx rm -rf src/generated/pgs && shx mkdir -p src/generated/pgs && gen-api-models --api-spec https://raw.githubusercontent.com/pagopa/pagopa-infra/main/src/core/api/payment_transactions_gateway_api/pgs-fe/v1/_openapi.json.tpl --no-strict --out-dir src/generated/pgs",
-    "generate:ts-from-yaml:ecommerce": "shx rm -rf src/generated/ecommerce && shx mkdir -p src/generated/ecommerce && gen-api-models --api-spec https://raw.githubusercontent.com/pagopa/pagopa-infra/main/src/domains/ecommerce-app/api/ecommerce-checkout/v1/_openapi.json.tpl --no-strict --out-dir src/generated/ecommerce"
+    "generate:ts-from-yaml:ecommerce": "shx rm -rf src/generated/ecommerce && shx mkdir -p src/generated/ecommerce && gen-api-models --api-spec https://raw.githubusercontent.com/pagopa/pagopa-infra/main/src/domains/ecommerce-app/api/ecommerce-checkout/v1/_openapi.json.tpl --no-strict --out-dir src/generated/ecommerce",
+    "generate:ts-from-yaml:ecommerce-v2": "shx rm -rf src/generated/ecommerce-v2 && shx mkdir -p src/generated/ecommerce-v2 && gen-api-models --api-spec https://raw.githubusercontent.com/pagopa/pagopa-infra/d8ad26e3b0c1f62e7afb0493ac5770b1015eab10/src/domains/ecommerce-app/api/ecommerce-checkout/v2/_openapi.json.tpl --no-strict --out-dir src/generated/ecommerce-v2"
```

```suggestion
    "generate:ts-from-yaml:ecommerce-v2": "shx rm -rf src/generated/ecommerce-v2 && shx mkdir -p src/generated/ecommerce-v2 && gen-api-models --api-spec https://raw.githubusercontent.com/pagopa/pagopa-infra/main/src/domains/ecommerce-app/api/ecommerce-checkout/v2/_openapi.json.tpl --no-strict --out-dir src/generated/ecommerce-v2"
```
To be merged after dependent infra PR merge